pkg/rpc/server/server.go

```diff
@@ -1937,7 +1909,7 @@ drainloop:
 func (s *Server) blockHeightFromParam(param *request.Param) (int, *response.Error) {
 	num, err := param.GetInt()
```

Comment:
Can we do `GetIntStrict()` here...

Reply:
We can't do `GetIntStrict()` here, because we have to correctly handle cases like `"params": ["1"]`, but we can reuse `blockHeightFromParam` in `blockHashFromParam`, so fixed.